src/pages/explore/[[...slug]].js

```diff
@@ -1,18 +1,38 @@
+import PropTypes from "prop-types";
 import React from "react";
 
 import ExplorePage from "@/pesayetu/components/ExplorePage";
+import TourProvider from "@/pesayetu/components/HURUmap/Tutorial";
```

Comment:
```suggestion
import Tutorial from "@/pesayetu/components/HURUmap/Tutorial";
```
Or why are we calling it `TourProvider` here?

Reply:
to easily show it's a provider at a glance. i think `TutorialProvider` works better @kilemensi

Reply:
Nah, if the component is called `Tutorial`, why would we want to change that name @KobbyMmo ?